engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -309,14 +309,16 @@ class WorkflowActor(val workflowId: WorkflowId,
     goto(finalState) using data.copy(currentLifecycleStateActor = None)
   }
 
+  private[workflow] def makeFinalizationActor(workflowDescriptor: EngineWorkflowDescriptor, executionStore: ExecutionStore, outputStore: OutputStore) = {
+    context.actorOf(WorkflowFinalizationActor.props(workflowId, workflowDescriptor, executionStore, outputStore, stateData.initializationData), name = s"WorkflowFinalizationActor-$workflowId")
```

Comment:
Dunno. This is a copy/paste from what it was before. Not too keen on doing half of a "rationalise actor naming" ticket in this PR